pycobertura/__main__.py

```diff
@@ -0,0 +1,8 @@
+import click
+
+from .cli import pycobertura
+
+cli = click.CommandCollection(sources=[pycobertura])
+
+if __name__ == "__main__":
+    cli()
```

Comment:
Can we add a test to ensure that this way of invoking pycobertura always works moving forward?

From the Python docs it seems that [runpy.run_module()](https://docs.python.org/3/library/runpy.html#runpy.run_module) provides the same functionality as `python -m` to be run from Python code.

Reply:
Yep, added a unit test